### ISSUE TITLE
lostFocusController: `suggestedElement` should be focusable

### ISSUE THREAD

**Issue by florian-sanders-cc** [CleverCloud/clever-components#1161]
## Context

The `lostFocusController` suggests elements to focus and when typechecking components that rely on this helper, @pdesoyres-cc suggested that it should always suggest a focusable element (not in the sense "an element that can receive focus" but at least "an element that exposes a focus method").

We need to check which types other than `HTMLElement` expose a `focus` method among types that inherit from `Element`.
